### ISSUE TITLE
Enchanting improvements

### ISSUE THREAD

**Issue by Innoxia** [Innoxia/liliths-throne-public#134]
1) I will add an enchant to remove parts. :3

2) I will add the ability to add extra slots to clothing soon! :3

I will definitely be adding a save/load template option to the enchanting system. :3

**Comment by bicobus** [comment]
Enchanting still remains a bit confusing, especially in the realm of numbers. If you craft a potion to increase size, the description say "add +15 size to boobs", but won't inform the player of the expected effect. To be more informative, the tooltip should say "will change my boobs from AA to DD".

Same thing with fluid production: how much is a lot? Like Penn Jillette once said: [I understand 3, but 9 is a bunch and 15 if a fucking lot](https://www.youtube.com/watch?v=Y0Z0raWIHXk).

**Comment by A-Master-of-Puppets** [comment]
It would be nice to have it say something like that, but it would need to say it for any NPCs in the tile as well; who says the potion I'm making is for me?

**Comment by bicobus** [comment]
> It would be nice to have it say something like that, but it would need to say it for any NPCs in the tile as well; who says the potion I'm making is for me?

Have a tooltip on the "drink me" button? What you are raising here is more of a UI issue though.

![javaw_2018-03-22_20-05-27](https://user-images.githubusercontent.com/26164751/37792569-66f571fe-2e0c-11e8-831e-3e5cc920bc51.png)
There is space on that screen to put a drop down menu to select for which target the potion's effect needs to be shown. Or you could work with the assumption that the player do not know the precise size of the other NPC in the tile.

**Comment by A-Master-of-Puppets** [comment]
Why would you assume that when the player _can_ get their exact sizes just by clicking on their name? Their genitals aren't immediately visible, but everything else about them is.

**Comment by bicobus** [comment]
How about this: by default the effects are listed for the player. But to get the effects for a NPC, the player needs to click on (select) that npc. The description should be updated too, saying something like "Effects for CHARACTERNAME".

> Why would you assume that when the player can get their exact sizes just by clicking on their name? Their genitals aren't immediately visible, but everything else about them is.

To increase the genitals size, one needs that kind of genitals first. So if the existence of specific parts are unknown, the detailed effects shouldn't be shown otherwise that would be telling.

**Comment by A-Master-of-Puppets** [comment]
If you click on an NPC you get their details, but it also kicks you out of the enchanting screen. I think there's enough room on the enchanting screen to have a list of NPCs in the cell, there should never be that many of them.

**Comment by bicobus** [comment]
Oh I see where misunderstanding lies. All along I was referring to the description of the end potion, not the process of making one.

If we are talking about the process, then what is shown should be generalistic. Give the player proper references, like "this potion would change a AA cup to D". I can mostly visualize what a AA cup is and what a DD cup is, so I can properly gauge the potency of the potion. If the potion would only tell me "+10 breasts size", that would leave me with nothing of value. I don't know how to interpret that data and I can't visualize the end result. What does +10 means anyway? There is no screen with numerical stats for bodyparts!

**Comment by bicobus** [comment]
I don't understand the functional difference between the different example you just wrote. The potency of the potion remains the same (+n for stat), what changes is how the information is conveyed to the player.

Mar 25, 2018, 11:23 AM by notifications@github.com:

**Comment by A-Master-of-Puppets** [comment]
Right, in your example, what is conveyed to the player is what effect +4 cup size would have on a _hypothetical_ AA cup NPC. In _my_ example, what is conveyed to the player is what effect that +4 cup size will have on the _actual_ NPC I want to use it on. One of these things I care about. One of these things I don't.
